docs/yearmonth.md

```diff
@@ -313,6 +313,26 @@ ym.difference(other, { largestUnit: 'months' })  // => P154M
 ym.withDay(1).difference(other.withDay(1), { largestUnit: 'days' });  // => P4687D
 ```
 
+### yearMonth.**equal**(_other_: Temporal.YearMonth) : boolean
+
+**Parameters:**
+- `other` (`Temporal.YearMonth`): Another month to compare.
+
+**Returns:** `true` if `yearMonth` and `other` are equal, or `false` if not.
+
+Compares two `Temporal.YearMonth` objects for equality.
+
+The difference with `Temporal.YearMonth.compare` is that this function will return `false` if the two months are expressed in different calendar systems.
```

Comment:
Temporal.YearMonth.prototype.compare isn't as much of an analogy here because of the refDay field.  (I assume that this just performs a straight equality on the data members.)